ketchup/main.py

```diff
@@ -1,121 +1,231 @@
 import datetime
-
+import math
 import dateparser
 import datetime as dt
-from ketchup.luffy import LUFFY
 import ketchup.watch_schedule_builder as wsb
-import ketchup.fun_fact as fun_fact
 
-DDP = dateparser.date.DateDataParser(languages=["en"], settings={"DATE_ORDER": "DMY"})
+# from ketchup.luffy import LUFFY
+# import ketchup.fun_fact as fun_fact
+
+DDP = dateparser.date.DateDataParser(
+    languages=["en"], settings={"DATE_ORDER": "DMY"})
 
 
 def get_date(
-    prompt: str, require_future: bool = False, require_past: bool = False
+        prompt: str, require_future: bool = False, require_past: bool = False
 ) -> datetime.datetime:
     while True:
         try:
             user_date = input(prompt)
             user_date = DDP.get_date_data(user_date).date_obj
         except ValueError:
-            print("Sorry, please only enter a date. an example would be '10 Aug 2023' ")
+            print("Sorry, please only enter a date. "
+                  "An example would be '10 Aug 2023' ")
             continue
         if require_future:
             if user_date.date() < dt.date.today():
-                print("Sorry, please only enter a future date")
+                print("Sorry, please only enter a future date.")
                 continue
         if require_past:
             if user_date.date() > dt.date.today():
-                print("Sorry, please only enter a past date")
+                print("Sorry, please only enter a past date.")
                 continue
         else:
             break
     return user_date
 
 
+def calculate_episodes_from_hours(episodes, episode_duration_mins=24):
+    total_hours = (episodes * episode_duration_mins) / 60
+    days = int(total_hours // 24)
+    hours = int(total_hours % 24)
+    return days, hours
+
+
 def run():
+    print("Hello and welcome to AnimeKetchup "
+          "for all your episode catch-up needs!")
     while True:
         try:
-            episodes = int(
-                input(
-                    "Hello! How many episodes are in the show you are interested in watching?: "
-                )
-            )
+            user_option = int(input(
```

Comment:
Update these to only show available options or mark them as (unimplemented)

Reply:
Resolved, added a TODO at the top of the file.